.travis/ghcup.sh

```diff
@@ -4,3 +4,8 @@ ghcup install $GHC_VER
 ghcup install-cabal $CABAL_VER
 ghcup set $GHC_VER
 cabal update
+echo "Cabal version before upgrade:"
+cabal --version
+cabal install cabal-install-2.4.0.0
+echo "Cabal version after upgrade:"
+cabal --version
```

Comment:
Hi @TristanCacqueray I think we're already installing `cabal-2.4`. We specify the version [here](https://github.com/stackbuilders/dotenv-hs/blob/master/.travis.yml#L14-L16). So, I don't think these changes are necessary.